HotelBooking.API/appsettings.json

```
@@ -1,9 +1,13 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "ConnectionStrings": {
+        "Default": "Server=DD-YENNTH1\\SQLEXPRESS;Database=HotelBooking;Trusted_Connection=True;TrustServerCertificate=True;"
```

```suggestion
        "Default": "Server=.\SQLEXPRESS;Database=HotelBooking;Trusted_Connection=True;TrustServerCertificate=True;"
```
.\SQLEXPRESS will work for all locally.

---

Do you mean .\\SQLEXPRESS?